content/registry/packages/aiven/api-docs/_index.md

```diff
@@ -16,95 +16,95 @@ A Pulumi package for creating and managing Aiven cloud resources.
-    <li><a href="kafkaconnector" title="KafkaConnector"><span class="symbol resource"></span>KafkaConnector</a></li>
-    <li><a href="kafkamirrormaker" title="KafkaMirrorMaker"><span class="symbol resource"></span>KafkaMirrorMaker</a></li>
-    <li><a href="kafkaschema" title="KafkaSchema"><span class="symbol resource"></span>KafkaSchema</a></li>
-    <li><a href="kafkaschemaconfiguration" title="KafkaSchemaConfiguration"><span class="symbol resource"></span>KafkaSchemaConfiguration</a></li>
-    <li><a href="kafkatopic" title="KafkaTopic"><span class="symbol resource"></span>KafkaTopic</a></li>
-    <li><a href="m3aggregator" title="M3Aggregator"><span class="symbol resource"></span>M3Aggregator</a></li>
-    <li><a href="m3db" title="M3Db"><span class="symbol resource"></span>M3Db</a></li>
-    <li><a href="mirrormakerreplicationflow" title="MirrorMakerReplicationFlow"><span class="symbol resource"></span>MirrorMakerReplicationFlow</a></li>
-    <li><a href="mysql" title="MySql"><span class="symbol resource"></span>MySql</a></li>
-    <li><a href="opensearch" title="OpenSearch"><span class="symbol resource"></span>OpenSearch</a></li>
-    <li><a href="opensearchaclconfig" title="OpenSearchAclConfig"><span class="symbol resource"></span>OpenSearchAclConfig</a></li>
-    <li><a href="opensearchaclrule" title="OpenSearchAclRule"><span class="symbol resource"></span>OpenSearchAclRule</a></li>
-    <li><a href="pg" title="Pg"><span class="symbol resource"></span>Pg</a></li>
-    <li><a href="project" title="Project"><span class="symbol resource"></span>Project</a></li>
-    <li><a href="projectuser" title="ProjectUser"><span class="symbol resource"></span>ProjectUser</a></li>
-    <li><a href="projectvpc" title="ProjectVpc"><span class="symbol resource"></span>ProjectVpc</a></li>
-    <li><a href="provider" title="Provider"><span class="symbol resource"></span>Provider</a></li>
-    <li><a href="redis" title="Redis"><span class="symbol resource"></span>Redis</a></li>
-    <li><a href="service" title="Service"><span class="symbol resource"></span>Service</a></li>
-    <li><a href="serviceintegration" title="ServiceIntegration"><span class="symbol resource"></span>ServiceIntegration</a></li>
-    <li><a href="serviceintegrationendpoint" title="ServiceIntegrationEndpoint"><span class="symbol resource"></span>ServiceIntegrationEndpoint</a></li>
-    <li><a href="serviceuser" title="ServiceUser"><span class="symbol resource"></span>ServiceUser</a></li>
-    <li><a href="transitgatewayvpcattachment" title="TransitGatewayVpcAttachment"><span class="symbol resource"></span>TransitGatewayVpcAttachment</a></li>
-    <li><a href="vpcpeeringconnection" title="VpcPeeringConnection"><span class="symbol resource"></span>VpcPeeringConnection</a></li>
+    <li><a href="account" title="Account"><span class="api-symbol api-symbol--resource"></span>Account</a></li>
```

Comment:
Maybe the docs gen template has an extra dash producing the value `api-symbol--resource`? Or is that intentional?

Reply:
Oh it does seem to be intentional? I am looking at https://github.com/pulumi/theme/pull/26/files#diff-ff21dd33fd708d1af5dbfe44512265a9bf974337ef988293e8e9765257506b96L50.

Reply:
BEM, I’m guessing? http://getbem.com/naming/